.github/workflows/zap_scan.yml

```diff
@@ -19,3 +19,17 @@ jobs:
         with:
           target: "https://wps-prod.pathfinder.gov.bc.ca/"
           rules_file_name: ".zap/rules.tsv"
+  zap_scan:
```

Comment:
Job "id" (in this case zap_scan) - is supposed to be unique: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id